lib/game_presenter.rb

```diff
@@ -0,0 +1,70 @@
+class GamePresenter
+    attr_accessor :presenter
+
+    def initialize(presenter:) 
+        @presenter = presenter
+    end
+
+    def get_input(input: nil)
+        presenter.get_input(input: input)
+    end
+
+    def output_message(message:)
```

Comment:
I assume you are using keyword arguments so that the methods are extra clear where they are used, but I'm not sure whether it's helpful when the method takes a single argument and the name of the method is already descriptive.